### DISCUSSION TITLE
[Bug]: sku in order list

### DISCUSSION THREAD

**Discussion by pmcr234** [pimcore/pimcore#17619] [Q&A / Support]
### Pimcore version

v6.6.1

### Steps to reproduce

Hey I want to export my order data with the sku, however that doesnt work. how do I do it? please help

### Actual Behavior

No data is shown, when I add "items" via Operator any getter, no info is being shown.

### Expected Behavior

Need the sku to be shown

**Comment by siwa-ddiesenreither** [comment]
@pmcr234 as for Pimcore v6.6.1 please read this - https://pimcore.com/en/resources/blog/pimcore-x-prepare-and-schedule-update_a148335

Could you also provide a bit more detail on how your order data is stored? Are you using Coreshop, the E-Commerce Framework, or a custom solution?

By the way, this might not be the best place for these technical questions, but I'm happy to help as much as I can!

**Comment by pmcr234** [comment]
Hey thanks for your reply! We are using coreshop. here is a screenshot of what I see in pimcore!

<img width="809" alt="Bildschirmfoto 2024-09-16 um 17 30 52" src="https://github.com/user-attachments/assets/21794ebb-c1a7-4e0b-a010-e01fe8df3f57">

**Reply by siwa-ddiesenreither** [reply]
![Bildschirmfoto 2024-09-16 um 16 02 38](https://github.com/user-attachments/assets/523ee935-011a-47a8-8190-d24f405040b0)

**Reply by pmcr234** [reply]
hey thanks for the help already, I did as the screenshot shows, however it only shows "object".

<img width="544" alt="Bildschirmfoto 2024-09-17 um 02 08 22" src="https://github.com/user-attachments/assets/2ba70728-45ed-4343-a3e5-b54197fb8d3c">

**Reply by siwa-ddiesenreither** [reply]
So how does your OrderItem Class look like and how does your Product Class look like?

**Reply by pmcr234** [reply]
sorry I am not aware what the class looks like, I didn't build the shop, can I provide you with any further screenshots?

**Reply by pmcr234** [reply]
Is it otherwise possible to show the object id in both product and order lists? I could match it after. My shop provider says that the sku is "too deep" inside the product object to show it directly. I dont know what that means and why that is the case.

<img width="821" alt="Bildschirmfoto 2024-09-17 um 16 18 43" src="https://github.com/user-attachments/assets/41f77d14-42bd-4043-a27c-288fc47f49fa">
<img width="1487" alt="Bildschirmfoto 2024-09-17 um 16 16 44" src="https://github.com/user-attachments/assets/8798d1c3-f674-42f0-9b9d-1d53b0b8e092">

**Reply by siwa-ddiesenreither** [reply]
Can you share a Screenshot of the 'Klassenattribute' of OrderItem and of CoreshopProduct. I think "too deep" is BS because in my example shown above i am selecting all items and then the product of an item and then the sku of the product - so the depth is irrelevant.

**Reply by pmcr234** [reply]
Hey okay that's already good intel. Here are the screenshots, I put numbers at the end to show the right sequence.
<img width="593" alt="order 2" src="https://github.com/user-attachments/assets/44d0f551-e34b-459c-a777-b020e5e1903d">
<img width="597" alt="order item 1" src="https://github.com/user-attachments/assets/d25b27da-a7fa-489b-82c0-46861b60c607">
<img width="605" alt="coreshop product 3" src="https://github.com/user-attachments/assets/a1b2e8b7-e265-40f2-bccd-f99f6dbed05e">
<img width="598" alt="coreshop product 2" src="https://github.com/user-attachments/assets/c60eed64-4572-4c93-89ed-971f583f9bb1">
<img width="605" alt="coreshop product 1" src="https://github.com/user-attachments/assets/2a098c17-4b57-4145-85c5-539dbc17d849">

**Reply by pmcr234** [reply]
if you need more screenshots just let me know :) thanks again

**Reply by pmcr234** [reply]
hey any update? :)
